dashboard/controller/stats.js

```diff
@@ -60,7 +60,7 @@ function getHowUserLaunchActivity(req, res) {
 			var d = {}
 			for (var i = 0; i < data.length; i++) {
 				if (!d.hasOwnProperty(data[i].event_object)) {
-					d[data[i].event_object] = 0
+					d[data[i].event_object] = 1
```

Comment:
If you set it to 1, then it'll generate the chart, but it'll show incorrect data. It'll show some has already launched one activity from the `List View` and one from the `Journal View`.

Reply:
I've doubled checked and it's working correctly for that case. Can you please elaborate, maybe some screenshots?